.github/workflows/scripts/download-nvd.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
In some non-Linux envs `bash` might not be available in `/bin`. To accommodate these outliers, use:

```suggestion
#!/usr/bin/env bash
```